src/app/modal-dialog/modal-dialog.demo.ts

```diff
@@ -30,7 +30,7 @@ export class ModalDialogDemoComponent {
   public closeResult: string;
 
   public title = 'Example Modal Dialog';
-  public isAlert = true;
+  public isAlert = false;
```

Comment:
https://github.com/infor-design/enterprise-ng/issues/432#issuecomment-483993245 - Does tihs solve this issue or do we need to update the demo as well. If so maybe just update the demo as well.

Reply:
Yes - this solves the issue in the demo.